airbyte-integrations/connectors/source-xero/source_xero/manifest.yaml

```
@@ -0,0 +1,2823 @@
+    name: "users"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/users_schema"
+    retriever:
+      $ref: "#/definitions/retriever"
+      record_selector:
+        $ref: "#/definitions/selector"
+        record_filter:
+          condition: "{{ record['UpdatedDateUTC'] >= stream_interval['start_time'] }}"
+    $parameters:
+      primary_key: "UserID"
+      path: "/Users"
+      extractor_path: "Users"
+    incremental_sync:
+      $ref: "#/definitions/incremental_sync_base"
+
+  branding_themes_stream:
+    $ref: "#/definitions/base_stream"
+    name: "branding_themes"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/branding_themes_schema"
+    $parameters:
+      primary_key: "BrandingThemeID"
+      path: "/BrandingThemes"
+      extractor_path: "BrandingThemes"
+
+  contact_groups_stream:
+    $ref: "#/definitions/base_stream"
+    name: "contact_groups"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/contact_groups_schema"
+    $parameters:
+      primary_key: "ContactGroupID"
+      path: "/ContactGroups"
+      extractor_path: "ContactGroups"
+
+  currencies_stream:
+    $ref: "#/definitions/base_stream"
+    name: "currencies"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/currencies_schema"
+    $parameters:
+      primary_key: "Code"
+      path: "/Currencies"
+      extractor_path: "Currencies"
+
+  organisations_stream:
+    $ref: "#/definitions/base_stream"
+    name: "organisations"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/organisations_schema"
+    $parameters:
+      primary_key: "OrganisationID"
+      path: "/Organisation"
+      extractor_path: "Organisations"
+
+  repeating_invoices_stream:
+    $ref: "#/definitions/base_stream"
+    name: "repeating_invoices"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/repeating_invoices_schema"
+    $parameters:
+      primary_key: "RepeatingInvoiceID"
+      path: "/RepeatingInvoices"
+      extractor_path: "RepeatingInvoices"
+
+  tax_rates_stream:
+    $ref: "#/definitions/base_stream"
+    name: "tax_rates"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/tax_rates_schema"
+    $parameters:
+      primary_key: "Name"
+      path: "/TaxRates"
+      extractor_path: "TaxRates"
+
+  tracking_categories_stream:
+    $ref: "#/definitions/base_stream"
+    name: "tracking_categories"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/tracking_categories_schema"
+    $parameters:
+      primary_key: "TrackingCategoryID"
+      path: "/TrackingCategories"
+      extractor_path: "TrackingCategories"
+
+  line_items_schema:
+    type:
+      - "null"
+      - object
```

Please change it to `type object ` for every stream schema, because actual declaration causes passing `$parameters ` into schema.
```suggestion
    type: object
```